tests/test_cases/runtime/test_device_macro_property.cu

```diff
@@ -87,10 +87,12 @@ TEST(DeviceMacroPropertyTest, DISABLED_ReadWrite) {
 
 FLAMEGPU_AGENT_FUNCTION(Init_add, flamegpu::MessageNone, flamegpu::MessageNone) {
     FLAMEGPU->environment.getMacroProperty<unsigned int>("int").exchange(1);
+    FLAMEGPU->environment.getMacroProperty<double>("double").exchange(1);
     return flamegpu::ALIVE;
 }
 FLAMEGPU_AGENT_FUNCTION(Write_add, flamegpu::MessageNone, flamegpu::MessageNone) {
     FLAMEGPU->environment.getMacroProperty<unsigned int>("int") += FLAMEGPU->getVariable<unsigned int>("a");
+    FLAMEGPU->environment.getMacroProperty<double>("double") += static_cast<double>(FLAMEGPU->getVariable<unsigned int>("a"));
```

Comment:
If you're going to update the test, atleast update it fully so the result is checked too.